l10n_ro_config/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2014 Forest and Biomass Romania
+# Copyright (C) 2020 NextERP Romania
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Romania - Localization Config",
+    "summary": "Romania - Localization Install and Config Applications",
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "author": "NextERP Romania,"
+    "Forest and Biomass Romania,"
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/l10n-romania",
+    "depends": ["l10n_ro", "l10n_ro_partner_create_by_vat"],
```

Comment:
De ce trebuie sa depinda de  l10n_ro_partner_create_by_vat ?

Reply:
Ca am mutat aici modif de header și footer, cu verificare vat_subjected.

Reply:
Propun sa mutam câmpurile aferente partenerului in acest modul. Iar modulul l10n_ro_partner_create_by_vat sa depindă de l10n_ro_config.